types/constants.go

```diff
@@ -39,6 +45,11 @@ var (
 	// The GenesisID is used in many places. Calculating it once saves lots of
 	// redundant computation.
 	GenesisID BlockID
+
+	// Oak hardfork constants.
+	OakHardforkBlock     BlockHeight
+	OakDifficultyMaxRise = big.NewRat(1004, 1000)
+	OakDifficultyMaxDrop = big.NewRat(1000, 1004)
```

Comment:
I'm going to leave it in `types` because that's where all of the other consensus tuning parameters are. Makes sense to me to keep everything together. I'll clean up the comments at least though.